From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matt Carroll <matthew-carroll@users.noreply.github.com>
Date: Sun, 3 Nov 2024 14:49:01 -0800
Subject: Added gclient files to gitignore to support Flock and other Flutter
 forks.

diff --git a/.gitignore b/.gitignore
index d43cb65efb1a01561b05f7b09bbe4b050301c1ed..d263bdb99b19030dd93dd513aa08cdb42c145a1f 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,8 @@
+# Injected by Flock to support custom forks
+.gclient
+.gclient_entries
+.gclient_previous_sync_commits
+
 # Do not remove or rename entries in this file, only add new ones
 # See https://github.com/flutter/flutter/issues/128635 for more context.
 
