From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matt Carroll <matthew-carroll@users.noreply.github.com>
Date: Sun, 26 Jan 2025 19:17:01 -0800
Subject: [Flock] - Alter update_engine_version.sh to use the latest Flutter
 commit as tagged by Flock during the daily rebase.


diff --git a/bin/internal/update_engine_version.sh b/bin/internal/update_engine_version.sh
index a35fa190aa2fecdfc3a066459229faa113b9ffaf..21e5315732a1ff1cc5f7f81a1388df8d2dc7f019 100755
--- a/bin/internal/update_engine_version.sh
+++ b/bin/internal/update_engine_version.sh
@@ -51,7 +51,10 @@ if [ -z "$ENGINE_VERSION" ] && [ -f "$FLUTTER_ROOT/DEPS" ] && [ -f "$FLUTTER_ROO
       ENGINE_VERSION=$(git -C "$FLUTTER_ROOT" merge-base HEAD origin/master)
     fi
   else
-    ENGINE_VERSION=$(git -C "$FLUTTER_ROOT" rev-parse HEAD)
+    # FLOCK: For the engine version, use the git commit hash for the most recent Flutter
+    #        commit, instead of the most recent Flock commit.
+    # ENGINE_VERSION=$(git -C "$FLUTTER_ROOT" rev-parse HEAD)
+    ENGINE_VERSION=$(git -C "$FLUTTER_ROOT" rev-parse flock-engine-hash)
   fi
 fi
 
